app/ts-sql/sql/server.go

```diff
@@ -170,6 +175,7 @@ func NewServer(conf config.Config, cmd *cobra.Command, logger *Logger.Logger) (a
 	s.castorService = castor.NewService(c.Analysis)
 	s.sherlockService = sherlock.NewService(c.Sherlock)
 	s.sherlockService.WithLogger(s.Logger)
+	s.continuousQueryService = continuousquery.NewService(time.Duration(c.ContinuousQuery.RunInterval))
```

Comment:
`continuousquery.NewService()` had been called twice. So, `s.continuousQueryService.QueryExecutor` is not right. For example, `s.continuousQueryService.QueryExecutor.StatementExecutor` is nil. This will lead to many problems.

Reply:
Yes, thx bro.